src/app/src/About.js

```diff
@@ -1,47 +1,59 @@
 import Carousel from 'nuka-carousel';
 import { PagingDots } from 'nuka-carousel/lib';
 import { func, number } from 'prop-types';
-import React, { Component } from 'react';
+import React from 'react';
 import { connect } from 'react-redux';
+import styled from 'styled-components';
 
 import { saveAboutSlideIndex } from './actions';
 
 // Amounts greater than 1 indicate how much of the surrounding slides to show
 const slidesToShow = 1.3;
 
-class About extends Component {
-    render() {
-        const { dispatch, aboutSlideIndex } = this.props;
-
-        return (
-            <div className='about'>
-                <h2>About</h2>
-                <Carousel
-                    autoplay={false}
-                    cellAlign={'center'}
-                    renderCenterLeftControls={null}
-                    renderCenterRightControls={null}
-                    renderBottomCenterControls={null}
-                    renderTopCenterControls={props => <PagingDots {...props} />}
-                    slideIndex={aboutSlideIndex}
-                    slidesToShow={slidesToShow}
-                    afterSlide={slideIndex =>
-                        dispatch(saveAboutSlideIndex(slideIndex))
-                    }
-                >
-                    <div className='about__slide'>The Fishway! So cool!!</div>
-                    <div className='about__slide'>
-                        Marine biologists! Even cooler!
-                    </div>
-                    <div className='about__slide'>
-                        Computer Engineers! Pretty cool too!
-                    </div>
-                    <div className='about__slide'>Dams! The coolest!</div>
-                </Carousel>
-            </div>
-        );
+const StyledAbout = styled.div`
+    padding-top: 1rem;
+
+    > h2 {
+        text-align: center;
     }
-}
+
+    .about__slide {
+        height: 80vh;
+        padding: 1rem;
+    }
+`;
+
+const About = props => {
```

Comment:
@maurizi pointed out that the class-based React Component doesn't need to be converted into a functional component.

https://www.styled-components.com/docs/advanced#refs

Reply:
That's more of an FYI, I don't think any changes need to be made because the `About` component won't need to rely on any React lifecycle methods or rerender, afaik.

Reply:
If we could put props in parens, that would be cleaner though `(props)`

Reply:
Ooh, gotcha! Thanks. I'm also happy to change it back if that's preferable. 

The linter changed it from `(props)` actually 🤔

Reply:
Ah ok, then this is fine :+1: